assets/components/directDebit/directDebitForm/directDebitForm.jsx

```diff
@@ -33,10 +41,13 @@ type PropTypes = {
   updateAccountNumber: (accountNumber: string) => void,
   updateAccountHolderName: (accountHolderName: string) => void,
   updateAccountHolderConfirmation: (accountHolderConfirmation: boolean) => void,
-  payDirectDebitClicked: (callback: Function) => void,
   openDDGuaranteeClicked: () => void,
   closeDDGuaranteeClicked: () => void,
   formError: string,
+  phase: string,
```

Comment:
If the `phase` is predetermined, we could create a type with the following shape:
```
type Phase = 'ENTRY' | 'CONFIRMATION' | ...
```
Then use it as the following:
```
phase: Phase,
```
In that way, we will be gaining extra checks in "compile" time.